.github/workflows/ci.yaml

```diff
@@ -1728,33 +1728,31 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
       - run: ./tools/ci.sh
-  # cactus-plugin-satp-hermes:
-  #   continue-on-error: false
-  #   env:
-  #     FULL_BUILD_DISABLED: true
-  #     JEST_TEST_PATTERN: packages/cactus-plugin-satp-hermes/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts
-  #     JEST_TEST_RUNNER_DISABLED: false
-  #     TAPE_TEST_RUNNER_DISABLED: true
-  #   needs: build-dev
-  #   runs-on: ubuntu-20.04
-  #   steps:
-  #     - name: Use Node.js ${{ env.NODEJS_VERSION }}
-  #       uses: actions/setup-node@v3.6.0
-  #       with:
-  #         node-version: ${{ env.NODEJS_VERSION }}
-  #     - uses: actions/checkout@v3.5.2
-  #     - id: yarn-cache-dir-path
-  #       name: Get yarn cache directory path
-  #       run: echo "::set-output name=dir::$(yarn cache dir)"
-  #     - id: yarn-cache
-  #       name: Restore Yarn Cache
-  #       uses: actions/cache@v3.3.1
-  #       with:
-  #         key: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
-  #         path: ./.yarn/
-  #         restore-keys: |
-  #           ${{ runner.os }}-yarn-
-  #     - run: ./tools/ci.sh
+  cactus-plugin-satp-hermes:
+    continue-on-error: false
+    env:
+      FULL_BUILD_DISABLED: true
+      JEST_TEST_RUNNER_DISABLED: false
+      JEST_TEST_PATTERN: packages/cactus-plugin-satp-hermes/src/test/typescript/(unit/.*/*.test.ts|integration/(client-.*.test.ts|odap.test.ts|odap-api.*.test.ts|server-.*.test.ts|backup-.*.test.ts|odap-rollback-.*.test.ts))
+      TAPE_TEST_RUNNER_DISABLED: true
+    needs: build-dev
+    runs-on: ubuntu-20.04
```

Comment:
@AndreAugusto11 Sorry, one more thing: please upgrade to 22.04 (we have a pr on the upstream repo that mass upgrades everything to 22.04)

Reply:
Thanks @petermetz!